web-registry/.env.example

```diff
@@ -11,8 +11,6 @@ REACT_APP_LEDGER_CHAIN_ID=regen-redwood-1
 REACT_APP_LEDGER_CHAIN_NAME="Regen - Redwood Testnet"
 REACT_APP_BLOCK_EXPLORER=https://redwood.regen.aneka.io
 REACT_APP_API_URI=http://localhost:5000
-# This is a feature flag to show/hide NCT basket related link in the header menu/footer.
-REACT_APP_NCT_BASKET=
```

Comment:
decided to rm the feature flag completely since I don't think we'll want to disable nct again in the future...